src/test/resources/features/Section2/Clearance/ClearanceJourney.feature

```diff
@@ -0,0 +1,176 @@
+@Regression
+Feature: Section2 Clearance Prelodged
+
+  Background:
+    Given I clear data in cache
+  #The initial section 2 Journey starts with below combination
+  #isEIDR=NO, #isExporter=NO, isExporterEORI=NO, holdTheContract=NO, carrierEoriNumber=No
+
+  @Section2 @ClearancePrelodgedJourney
+  Scenario: Exports Clearance Prelodged Scenario When the user is not having
+  entry in declarant records is not an exporter,not having EORI, not holding
+  the contract to submit the declaration on behalf and not having the carrier EORI number
+    Given I fill section1 for CLEARANCE,prelodged declaration
+    Then I should land on Entry-Into-Declarant-Records page
+    And I select No to is this an entry into declarant records
+    And I click continue
+    Then I should land on Declarant-Details page
+    And I select Yes to confirm my eori
+    And I click continue
+    Then I should land on Are-You-The-Exporter page
+    And I select No to I am the Exporter
+    And I click continue
+    Then I should land on Exporter-Eori-Number page
+    And I select No on exporter eori number page
+    And I click continue
+    Then I should land on Exporter-Address page
+    And I provide Exporter Address Details
+    And I click continue
+    Then I should land on Is-This-Exs page
+    And I select Yes to is this exs
+    And I click continue
+    Then I should land on Consingor-EORI-Number page
+    And I select No to consignor eori number
+    And I click continue
+    Then I should land on Consingor-Address page
+    And I provide consignor details
+    And I click continue
+    Then I should land on Third-Party-Goods-Transportation page
+    And I select No on third party goods transportation page
+    And I click continue
+    Then I should land on Consignee-Details page
+    And I provide consignee details
+    And I click continue
+    Then I should land on Is-Authorisation-Required page
+    And I select Yes to declare authorisations
+    And I click continue
+    And I select first Authorisation code ACR and enter eori as GB123456789008
+    And I click continue
+    Then I should land on Authorisations-Required-List page
+    And I select No to add another authorisation
+    And I click continue
+    Then I should land on MiniCYA-Section-2 page
+    And I check the MiniCYA page for Section-2
+    And I click continue on MiniCya
+
+    #  Given the user is on the Entry into declarant records page
```

Comment:
Looking at this comment is look like we have commented out the steps. please can you remove given, when , then words and leave rest of the comments

Reply:
Done